test/e2e/tests/test_ca_activation.py

```diff
@@ -80,7 +81,7 @@ def simple_certificate_authority():
     k8s.create_custom_resource(ca_ref, ca_resource_data)
     ca_cr = k8s.wait_resource_consumed_by_controller(ca_ref)
 
-    time.sleep(30)
+    time.sleep(CREATE_CA_WAIT_AFTER_SECONDS)
```

Comment:
I was going to say it might make more sense to just poll on the resource state with a maxTimeout, but this is roughly the same thing, just with no resource polling.